project.godot

```diff
@@ -476,6 +476,11 @@ g_eject_engulfed={
 "events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":72,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
  ]
 }
+g_sprint={
+"deadzone": 0.5,
+"events": [ Object(InputEventKey,"resource_local_to_scene":false,"resource_name":"","device":0,"alt":false,"shift":false,"control":false,"meta":false,"command":false,"pressed":false,"scancode":16777237,"physical_scancode":0,"unicode":0,"echo":false,"script":null)
```

Comment:
What key is this? Would it be possible to make sprint be shift, like it is in most games?

Reply:
This is shift. From my testing it seems to work.

Reply:
Alright, that's great then :+1: